plugins/calendar/src/test/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarLogicTest.java

```diff
@@ -143,4 +150,21 @@ private static DateTime fromLocalDateTime(LocalDateTime localDateTime) {
 		return new DateTime(localDateTime.atZone(ZoneId.systemDefault()).toInstant().toEpochMilli());
 	}
 
+	/**
+	 * @param iso
+	 *            date-time without a time-zone
+	 * @return
+	 */
+	private static DateTime fromISO2(String iso) {
+		DateFormat df = new SimpleDateFormat("yyyy-MM-dd");
+		Date date = null;
+		try {
+			date = df.parse(iso);
+		} catch (ParseException e1) {
```

Comment:
don't catch exceptions in tests, exceptions should fail the test

Reply:
okay